### DISCUSSION TITLE
Micropython-lib Missing git version tag for `v1.21.1`

### DISCUSSION THREAD

**Discussion by Josverl** [micropython/micropython#13514] [General]
I noticed some errors thrown by my tools when trying to build stubs for v1.22.1.

I understood from prior conversations that from v1.20.0 micropython-lib would use the same tagging approach as the micropython repo, including micro versions and -preview as the micropython repo does.

Perhaps just an oversight  ?

repo: micropython-lib 
```
> git tag --sort=v:refname
v0.1
v0.5
v1.0
v1.8.2
v1.8.4
v1.8.6
v1.9
v1.9.3
v1.20.0
v1.21.0
v1.22.0
```
repo: micropython (trimmed) 
```
v1.15
v1.16
v1.17
v1.18
v1.19
v1.19.1
v1.20.0
v1.21.0
v1.22.0
v1.22.0-preview
v1.22.1
v1.23.0-preview
```

And yes, i know that its pinned as a git module as well. 
IMO hashes and guids are for computers, my wetware prefers tags :-)

**Comment by dpgeorge** [comment]
You are right, this was an oversight.

It's now fixed, `micropython-lib` has been tagged with `v1.22.1` at the commit that was submoduled in the `v1.22.1` release of `micropython`.
